js/app/interfaces/card.js

```diff
@@ -157,6 +160,18 @@ const Card = new Lang.Interface({
             GObject.ParamFlags.READWRITE,
             ''),
         /**
+         * Property: collection-name
+         * The name of whatever it is a set card contains
+         *
+         * For sets, we want to show how many items are contained within it.
+         * To do this we need to know the name of the things being contained,
+         * e.g. 'lessons' or 'articles'.
+         */
+        'collection-name': GObject.ParamSpec.string('collection-name',
```

Comment:
Discussed on hangout that we'll stick with hardcoded "item"/"items" for now, using `ngettext()`, and open a new issue for customizing this text, possibly through a YAML extension